oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsNetworkIF.java

```diff
@@ -208,6 +216,8 @@ public boolean updateAttributes() {
             this.collisions = ParseUtil.unsignedIntToLong(ifRow.dwOutDiscards); // closest proxy
             this.inDrops = ParseUtil.unsignedIntToLong(ifRow.dwInDiscards); // closest proxy
             this.speed = ParseUtil.unsignedIntToLong(ifRow.dwSpeed);
+            this.ifAlias = ""; // not supported by MIB_IFROW
+            this.ifOperStatus = IfOperStatus.UNKNOWN; // TODO find a mapping
```

Comment:
Don't think this is a TODO.  I think it's safe to ignore doing anything extraordinary for XP / Win Server 2003 and just say not supported.